bucket/oh-my-posh.json

```diff
@@ -3,15 +3,20 @@
     "description": "A prompt theme engine for any shell",
     "homepage": "https://ohmyposh.dev",
     "license": "MIT",
-    "notes": "Refer to 'https://ohmyposh.dev/docs/windows#replace-your-existing-prompt' for shell specific configurations.",
+    "notes": [
+        "Thanks for installing Oh My Posh.",
+        "Have a look at https://ohmyposh.dev/docs/installation/prompt for detailed instructions for your shell."
+    ],
```

Comment:
```suggestion
    "notes": "Refer to 'https://ohmyposh.dev/docs/installation/prompt' for shell specific configurations.",
```

Reply:
I originally copied the notes from the dev. But it is okay for simplicity.